.eslintrc

```diff
@@ -1,26 +1,28 @@
 {
-	"parser": "babel-eslint",
 	"env": {
 		"node": true
 	},
-	"ecmaFeatures": {
-		"arrowFunctions": true,
-		"blockBindings": true,
-		"classes": true,
-		"defaultParams": true,
-		"destructuring": true,
-		"forOf": true,
-		"modules": true,
-		"objectLiteralComputedProperties": true,
-		"objectLiteralShorthandMethods": true,
-		"objectLiteralShorthandProperties": true,
-		"spread": true,
-		"superInFunctions": true,
-		"templateStrings": true,
-		"unicodeCodePointEscapes": true,
-		"jsx": true
+	"parser": "babel-eslint",
```

Comment:
Only structure/formatting changes in this file. ESLint threw errors with the old config.